manifests/m/Microsoft/err/6.4.5/Microsoft.err.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.err
+PackageVersion: 6.4.5
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Error Lookup Tool
+License: All Rights Reserved
+Copyright: © Microsoft Corporation. All rights reserved.

```

```suggestion
License: Proprietary
Copyright: Copyright (c) Microsoft Corporation.
```
A copyright isn't a license